main.go

```diff
@@ -136,7 +141,12 @@ func main() {
 				CaAddress:      common.HexToAddress(caAddress),
 			}
 
-			rpcServer, _, _, err := rpc.InitChainServiceAndRunRpcServer(pkString, chainOpts, useDurableStore, durableStoreFolder, useNats, msgPort, rpcPort)
+			ss, err := fs.Sub(fs.FS(staticSite), "packages/nitro-gui/dist")
```

Comment:
Can this be controlled by a flag, so serving the nitro UI is optional?

Reply:
Yes [f2c7ed8](https://github.com/statechannels/go-nitro/pull/1449/commits/f2c7ed87b903d84a112241b13258e5860a415f33)

I don't think we can make embedding the UI optional, but we can make serving it optional. This is because the compiler will complain if we try to embed a directory with no matching files in it. 

If we really care about that, we can come up with a solution:
* Have a `build_fake_UI` script which just writes an empty or trivial html file into the `packages/nitro-gui/dist` directory. We can coordinate builds with a makefile and have a `make go-nitro/headless` command which calls that first.
* Or have a new folder which we check the trivial `index.html` into, and have the production gui build overwrite that folder with the full UI.

Reply:
Digging around in the `boost` repo, I believe they are just requiring their react frontend to be built before boost can be built (and living with it 🤷 ).

Reply:
Here's a sketch at something we could do https://github.com/statechannels/go-nitro/pull/1452